build-test.cmd

```diff
@@ -360,6 +366,20 @@ if errorlevel 1 (
     exit /b 1
 )
 
+xcopy /s /y "%CORE_ROOT_STAGE%" "%CORE_ROOT%"
```

Comment:
Also to be clear, my first reply (https://github.com/dotnet/coreclr/pull/10979#discussion_r111642079) was talking about building tests on Windows for non-Windows platforms, not talking about building the product on non-Windows